### DISCUSSION TITLE
Hardware requirements for 2000 concurrent users

### DISCUSSION THREAD

**Discussion by abdullahqureshi1994** [mediacms-io/mediacms#703] [Q&A]
Hi All,

Can anyone suggest or give an idea regarding the hardware requirements for 2000 concurrent users for Media CMS deployed server?

**Comment by Anduin2017** [comment]
Assuming a video streaming rate of 4 Mbps per user, 2000 concurrent users require a total bandwidth of 8 Gbps (4 Mbps x 2000). However, this may not be sufficient to provide high-quality video streaming services. Therefore, it is recommended to conduct adequate measurement and planning of network traffic during deployment to ensure that the server can successfully handle this scale of concurrent users. It is also worth noting that network bandwidth is not always the only factor to consider for smooth playback and high-quality user experience. Server performance and network quality are equally important.
